integration_tests/structs_15.py

```diff
@@ -1,5 +1,6 @@
-from lpython import i32, i16, i8, i64, CPtr, dataclass, ccall, Pointer, c_p_pointer, sizeof
+from lpython import i32, i16, i8, CPtr, dataclass, ccall, Pointer, c_p_pointer, sizeof, ccallable
 
+@ccallable
```

Comment:
I feel that any ``dataclass`` which is being used in C code as well should be decorated with ``ccallable`` and ``lpython.py`` should raise an error in case we try to use a pure Python ``dataclass`` with C-APIs. What do you say @certik?

Reply:
Yes, I think so.